diagnostic.md

```diff
@@ -6,21 +6,21 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    You define the routes inside the router and the routes define the models
     ```
 
 1.  What is the command to generate a route named `boston` nested under
     `campus`?
 
     ```md
-    <!-- your response here -->
+    ember g route campus/boston
     ```
 
 1.  Suppose you have a nested route at the URL `/campus/boston`. How would you
     use the `link-to` helper to generate an appropriate link?
 
     ```md
-    <!-- your response here -->
+    {{#link-to 'campus'}}back{{/link-to}}
```

Comment:
We're looking for the link-to helper that would link us to `/campus/boston`, thus, `{{#link-to 'campus.boston'}}Boston{{/link-to}}`